audbackend/core/backend/base.py

```diff
@@ -135,6 +190,14 @@ def copy_file(
                 verbose,
             )
 
+            if validate:
+                self._assert_equal_checksum(
+                    dst_path,
+                    False,
+                    src_path,
+                    False,
+                )
```

Comment:
The source code is not so easy to read here, as it is not obvious what `False` means.
I guess the easiest fix would be to just write the actual variable names here as well:

```suggestion
                self._assert_equal_checksum(
                    path=dst_path,
                    path_is_local=False,
                    path_ref=src_path,
                    path_ref_is_local=False,
                )
```

If you agree with this, it needs to be changed at the other positions as well.

Reply:
Ok, I made `_assert_equal_checksum()` a key-word only function.